master/setup.py

```diff
@@ -208,7 +198,8 @@ def make_release_tree(self, base_dir, files):
         # alternative MySQL driver, that works under pypy
         'pymysql == 0.7.1',
         'PyJWT == 1.4.0',
-        'pynats == 0.0.1'
+        'pynats == 0.0.1',
+        'www'
```

Comment:
@christiankjaer about broken setup tools, maybe it makes sense to add  'setuptools >=  21.1.0' here, so we make sure the proper version is used